dev-vcs/bzr/bzr-2.7.0.recipe

```diff
@@ -0,0 +1,52 @@
+SUMMARY="A powerful distributed version control system"
+DESCRIPTION="Bazaar (bzr) is a version control system that helps you track \
+project history over time and to collaborate easily with others. Whether \
+you're a single developer, a co-located team or a community of developers \
+scattered across the world, Bazaar scales and adapts to meet your needs.
+Part of the GNU Project, Bazaar is free software sponsored by Canonical."
+HOMEPAGE="https://bazaar.canonical.com"
+COPYRIGHT="2005-2013 Canonical"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://launchpad.net/bzr/2.7/$portVersion/+download/bzr-$portVersion.tar.gz"
+CHECKSUM_SHA256="0d451227b705a0dd21d8408353fe7e44d3a5069e6c4c26e5f146f1314b8fdab3"
+SOURCE_DIR="bzr-$portVersion"
+
+if [ $effectiveTargetArchitecture = x86_gcc2 ]; then
+	PATCHES="bzr-2.7.0_gcc2.patchset"
+fi
```

Comment:
Shouldn't it be applied to x86_64 too? Specially the second part of it for the location to the SSL certificates?